packages/angular/projects/clarity-adoption/tsconfig.app.json

```diff
@@ -3,7 +3,13 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "outDir": "../../out-tsc/adoption",
-    "types": []
+    "types": [],
+    "paths": {
+      "@clr/angular": ["dist/clr-angular"],
+      "@clr/angular/*": ["dist/clr-angular/*"],
+      "@cds/angular": ["dist/angular"],
+      "@cds/angular/*": ["dist/angular/*"]
+    }
```

Comment:
Map the dist output so when the source code changes in the lib the dev apps auto reload